dynatrace/src/main/java/com/dynatrace/opentelemetry/metric/Serializer.java

```diff
@@ -31,9 +32,11 @@
       "Could not create metric line for data point with name %s (%s).";
 
   private final MetricBuilderFactory builderFactory;
+  private final CumulativeToDeltaConverter deltaConverter;
 
   Serializer(MetricBuilderFactory builderFactory) {
     this.builderFactory = builderFactory;
+    this.deltaConverter = new CumulativeToDeltaConverter(Duration.ofMinutes(15));
```

Comment:
I guess this should be configurable by the user somehow, no?

Reply:
We had an internal discussion about this, and decided on this as the default value. I guess we could always extend it later if we need to.